evm/src/cpu/kernel/interpreter.rs

```diff
@@ -423,13 +485,16 @@ impl<'a> Interpreter<'a> {
                 }
             }?;
         }
-        println!("Opcode count:");
-        for i in 0..0x100 {
-            if self.opcode_count[i] > 0 {
-                println!("{}: {}", get_mnemonic(i as u8), self.opcode_count[i])
+        #[cfg(test)]
```

Comment:
What we can do, similarly to #1503, is to use `#[cfg(debug_assertions)]` instead. This will not change anything when we run tests in `debug` mode, and can still display everything in release mode iif we enable debug assertions, so that if we want to print CPU simulation for some reason (during debugging), just using an environment variable suffices instead of changing manually the code.